tests/testit/test-datatables.R

```diff
@@ -192,3 +192,31 @@ assert('warn autoHideNavigation if no pageLength', {
     datatable(head(iris, 5), autoHideNavigation = TRUE, options = list(pageLength = 20))
   ))
 })
+
+assert("colDefsTgtHandle() works", {
+  cols = c("A", "B", "C")
+  (colDefsTgtHandle(NULL, cols) %==% list())
+  (colDefsTgtHandle(NULL, cols) %==% list())
+
+  defs = list(
+    list(1, targets = "_all"),
+    list(2, targets = 1L),
+    list(3, targets = "B"),
+    list(4, targets = c("A", "_all")),
+    list(5, targets = list(c("A", "C"), "_all")),
+    list(6, targets = list(1L, "_all")),
+    list(7, targets = list(1L, "C")),
+    list(8, targets = list(1L, "B", "_all"))
+  )
+  res = list(
+    list(1, targets = "_all"),
+    list(2, targets = 1L),
+    list(3, targets = 1L),
+    list(4, targets = list("_all", 0L)),
```

Comment:
I can't remember why it should return `list("_all", 0L)` instead of `list("_all")`. What's the difference??